iexec-result-proxy-library/build.gradle

```diff
@@ -22,7 +23,7 @@ testing {
         test {
             useJUnitJupiter()
             dependencies {
-                implementation 'org.junit.jupiter:junit-jupiter:5.8.2'
+                implementation 'org.junit.jupiter:junit-jupiter'
```

Comment:
Can be removed as well as its encapsulating `dependencies` block which becomes empty